main.go

```diff
@@ -0,0 +1,18 @@
+package main
```

Comment:
Can you make a short explanation about this program?

Reply:
do you mean, add a comment to the code or add a comment here, in the review?

Reply:
I was reading the code better and finally discovered the wheel.

The weirdness of the first sight with a new programming language.

I understand that the expected response from Arrebol was "mocked" (or stubbed?) on an HTTP server Go to use it in Py code tests.

I'll accept the changes.